.gitignore

```diff
@@ -12,7 +12,9 @@
 /Test*.html
 /Test*.json
 /Test*.log
+/Test*.log.gz
```

Comment:
we should probably also starter-kit this one and propogate this over to other projects :)

Reply:
Done